src/common/url-validator.ts

```diff
@@ -6,14 +6,19 @@ export class UrlValidator {
     public async isSupportedUrl(url: string, chromeAdapter: BrowserAdapter): Promise<boolean> {
         const lowerCasedUrl: string = url.toLowerCase();
         if (lowerCasedUrl.match('http://*/*') || lowerCasedUrl.match('https://*/*')) {
```

Comment:
While you're here, this regex looks like it's trying to test the wrong thing but happens to be working by accident (I bet it would fail a test case like `http:/oops` and would only accidentally pass a test case like `http://domain-with-no-trailing-slash`) :)

Reply:
probably accidentally passes `oops_http://example.com` too

Reply:
You're right - added these test cases and updated to fix.